dir.targets

```diff
@@ -166,14 +166,22 @@
 
   <Target Name="Test"
           DependsOnTargets="Build;EnsurePrerequisitesCopied"
-          Condition="'$(IsTestProject)' == 'true'">
+          Condition="'$(IsTestProject)' == 'true' and '$(RunTests)' != 'false'">
```

Comment:
I think I'll actually just remove this condition.  It's easily accomplished by just using the `Build` target instead of `BuildAndTest`.